{
	"name": "CVE-2021-47131",
	"threat_severity": "Moderate",
	"public_date": "2024-03-15T00:00:00Z",
	"bugzilla": {
		"description": "kernel: net/tls: Fix use-after-free after the TLS device goes down and up",
		"id": "2269820",
		"url": "https://bugzilla.redhat.com/show_bug.cgi?id=2269820"
	},
	"cvss3": {
		"cvss3_base_score": "5.5",
		"cvss3_scoring_vector": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H",
		"status": "draft"
	},
	"cwe": "CWE-416",
	"details": [
		"No description is available for this CVE."
	],
	"references": [
		"https://www.cve.org/CVERecord?id=CVE-2021-47131\nhttps://nvd.nist.gov/vuln/detail/CVE-2021-47131\nhttps://lore.kernel.org/linux-cve-announce/2024031513-CVE-2021-47131-eafc@gregkh/T/#u"
	],
	"package_state": [
		{
			"cpe": "cpe:/o:redhat:enterprise_linux:6",
			"fix_state": "Out of support scope",
			"package_name": "kernel",
			"product_name": "Red Hat Enterprise Linux 6"
		},
		{
			"cpe": "cpe:/o:redhat:enterprise_linux:7",
			"fix_state": "Out of support scope",
			"package_name": "kernel",
			"product_name": "Red Hat Enterprise Linux 7"
		},
		{
			"cpe": "cpe:/o:redhat:enterprise_linux:7",
			"fix_state": "Out of support scope",
			"package_name": "kernel-rt",
			"product_name": "Red Hat Enterprise Linux 7"
		},
		{
			"cpe": "cpe:/o:redhat:enterprise_linux:8",
			"fix_state": "Under investigation",
			"package_name": "kernel",
			"product_name": "Red Hat Enterprise Linux 8"
		},
		{
			"cpe": "cpe:/o:redhat:enterprise_linux:8",
			"fix_state": "Under investigation",
			"package_name": "kernel-rt",
			"product_name": "Red Hat Enterprise Linux 8"
		},
		{
			"cpe": "cpe:/o:redhat:enterprise_linux:9",
			"fix_state": "Under investigation",
			"package_name": "kernel",
			"product_name": "Red Hat Enterprise Linux 9"
		},
		{
			"cpe": "cpe:/o:redhat:enterprise_linux:9",
			"fix_state": "Under investigation",
			"package_name": "kernel-rt",
			"product_name": "Red Hat Enterprise Linux 9"
		}
	],
	"upstream_fix": "kernel 5.10.43, kernel 5.12.10, kernel 5.13",
	"csaw": false
}
